src/App.js

```diff
@@ -1,25 +1,51 @@
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import { Switch, Route } from 'react-router-dom';
+import  { Homepage }  from '../src/Homepage/Homepage.js';
+import  { WordSelector }  from '../src/WordSelector/WordSelector.js';
+import  { Gamepage }  from '../src/Gamepage/Gamepage.js';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export class App extends Component{
+  constructor() {
+    super()
+    this.state = {}
+  }
+
+  componentDidMount() {
+
+  }
+
+
+  render() {
+    return (
+      <div className="BangWords">
+        <header className="BangWords-header">
+          BangWords
+        </header>
+        <Switch>
+         <Route
+            exact path='/'
+            render={() => {
+            return  <Homepage/>
+            }}
+          />
+         <Route
+            exact path='/word-selector'
```

Comment:
I couldn't remember if you said this or not, but neither this nor the following Route need to have `exact` since they are in a switch and their relative paths are exclusive to one another. 

I guess I'm fine leaving them, but in larger projects, it not be best practice (though I'm not sure).  I could see it being unexpected or forgotten about to other contributors.

Reply:
![you lied to me!](https://media.giphy.com/media/12NlCFUvTokWXe/giphy.gif)